.github/workflows/crawler.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           bundle install
 
       - run: |
-          rm -f index*
+          [[ -n ${github.event.inputs+x} && "$github.event.inputs" == "true" ]] && rm -f index*
```

Comment:
Not all workflow use `checkbox` as input, some of them use text input and expect `latest` (in `relaton-data-iec`) of `force` (in `relaton-data-3gpp`) input

My proposal: because only for several repos `index*` should be deleted, let's do this in the corresponding `crawler.rb` and remove `rm -f index*` from common workflow